File: Dev/Android/YoopoonMarket/src/com/yoopoon/market/domain/ProductEntity.java
package com.yoopoon.market.domain;

import java.io.Serializable;

public class ProductEntity implements Serializable {
	/**
	 * @fieldName: serialVersionUID
	 * @fieldType: long
	 * @Description: TODO
	 */
	private static final long serialVersionUID = 1L;
	public int Id = 1;
	public int BussnessId = 0;
	public String BussnessName = null;
	public float Price = 0;
	public String Name;
	public int Status = 0;
	public String MainImg = null;
	public int IsRecommend = 0;
	public int Sort = 0;
	public int Stock = 0;
	public int Adduser = 0;
	public String Addtime;
	public int UpdUser = 0;
	public String UpdTime;
	public String Subtitte = null;
	public String Contactphone = null;
	public int Type = 0;
	public String TypeString;
	public String StatusString;
	public String NewPrice = null;
	public String Owner = null;
	public int CategoryId = 0;
	public String Detail = null;
	public String Img = null;
	public String Img1 = null;
	public String Img2 = null;
	public String Img3 = null;
	public String Img4 = null;
	public String SericeInstruction = null;
	public String Ad1 = null;
	public String Ad2 = null;
	public String Ad3 = null;
	public String ParameterValue = null;

	@Override
	public String toString() {
		return "ProductEntity [Id=" + Id + ", BussnessId=" + BussnessId + ", BussnessName=" + BussnessName + ", Price="
				+ Price + ", Name=" + Name + ", Status=" + Status + ", MainImg=" + MainImg + ", IsRecommend="
				+ IsRecommend + ", Sort=" + Sort + ", Stock=" + Stock + ", Adduser=" + Adduser + ", Addtime=" + Addtime
				+ ", UpdUser=" + UpdUser + ", UpdTime=" + UpdTime + ", Subtitte=" + Subtitte + ", Contactphone="
				+ Contactphone + ", Type=" + Type + ", TypeString=" + TypeString + ", StatusString=" + StatusString
				+ ", NewPrice=" + NewPrice + ", Owner=" + Owner + ", CategoryId=" + CategoryId + ", Detail=" + Detail
				+ ", Img=" + Img + ", Img1=" + Img1 + ", Img2=" + Img2 + ", Img3=" + Img3 + ", Img4=" + Img4
				+ ", SericeInstruction=" + SericeInstruction + ", Ad1=" + Ad1 + ", Ad2=" + Ad2 + ", Ad3=" + Ad3
				+ ", ParameterValue=" + ParameterValue + "]";
	}

}

File: Dev/Code/Community.Entity/Model/Product/ProductEntity.cs
using System;
using System.Collections.Generic;
using Community.Entity.Model.Category;
using Community.Entity.Model.ProductComment;
using Community.Entity.Model.ProductDetail;
using Community.Entity.Model.ProductParameter;
using YooPoon.Core.Data;

namespace Community.Entity.Model.Product
{
	public class ProductEntity : IBaseEntity
	{
		/// <summary>
		/// Id
		/// </summary>
		public virtual int Id { get; set; }
		/// <summary>
		/// 分类排序
		/// </summary>
		public virtual CategoryEntity Category { get; set; }
		/// <summary>
		/// 商家Id
		/// </summary>
		public virtual int BussnessId { get; set; }
		/// <summary>
		/// 商家名称
		/// </summary>
		public virtual string BussnessName { get; set; }
		/// <summary>
		/// 价格
		/// </summary>
		public virtual decimal Price { get; set; }
		/// <summary>
		/// 商品名称
		/// </summary>
		public virtual string Name { get; set; }
		/// <summary>
		/// 商品状态
		/// </summary>
		public virtual EnumProductStatus Status { get; set; }
		/// <summary>
		/// 商品主图
		/// </summary>
		public virtual string MainImg { get; set; }
		/// <summary>
		/// 商家推荐标识
		/// </summary>
		public virtual int IsRecommend { get; set; }
		/// <summary>
		/// 分类排序
		/// </summary>
		public virtual int Sort { get; set; }
		/// <summary>
		/// 库存计数
		/// </summary>
		public virtual int Stock { get; set; }
		/// <summary>
		/// AddUser
		/// </summary>
		public virtual int AddUser { get; set; }
		/// <summary>
		/// AddTime
		/// </summary>
		public virtual DateTime AddTime { get; set; }
		/// <summary>
		/// UpdUser
		/// </summary>
		public virtual int UpdUser { get; set; }
		/// <summary>
		/// UpdTime
		/// </summary>
		public virtual DateTime UpdTime { get; set; }
		/// <summary>
		/// 副标题
		/// </summary>
		public virtual string Subtitte { get; set; }
		/// <summary>
		/// 联系电话
		/// </summary>
		public virtual string Contactphone { get; set; }
		/// <summary>
		/// 商品类型
		/// </summary>
		public virtual EnumProductType Type { get; set; }
		/// <summary>
		/// 商品明细
		/// </summary>
		public virtual ProductDetailEntity Detail { get; set; }
		/// <summary>
		/// 商品评论
		/// </summary>
		public virtual List<ProductCommentEntity> Comments { get; set; }
		/// <summary>
		/// 商品属性
		/// </summary>
		public virtual List<ProductParameterEntity> Parameters { get; set; }
        /// <summary>
        /// 原价
        /// </summary>
        public virtual decimal? OldPrice { get; set; }
        /// <summary>
        /// 有多少人抢购
        /// </summary>
        public virtual int? Owner { get; set; }
	}
}